ChatSX/components/InputBox/models/LaunchActivity.ts

```diff
@@ -0,0 +1,6 @@
+import {MediaType} from 'react-native-image-picker';
+
+export interface LaunchActivity {
```

Comment:
rename to `LaunchActivityProps`, also when we don't have activities, we should name these to more appropriate name.